### ISSUE TITLE
Intel Raid relay starting with half health

### ISSUE THREAD

**Issue by jmartinez989** [long-war-2/lwotc#309]
Resistance relay started with half health as seen below. This does not seem to happen all of the time because all of the other intel raids I have done so far the relay spawned with the correct health.

[save_Half Health Relay.zip](https://github.com/long-war-2/lwotc/files/3365689/save_Half.Health.Relay.zip)

![20190707093408_1](https://user-images.githubusercontent.com/16939345/60769231-391ba980-a09b-11e9-96f0-bf87538693e3.jpg)

**Comment by pledbrook** [comment]
It looks like the HP bar is displaying the vanilla number of max HP pips, but the current "health" is the new starting HP using the LW2 numbers.

**Comment by MiserablePileOfBugs** [comment]
I had the same bug in my recent Liberation 3 mission. The relay had the correct HP because it took quite a few shots to destroy it but it displayed only a few pips of health. I have included a save below.

Curiously enough, when I did a Dark Event counter mission afterwards, the relay displayed the right HP.

[save_relay.zip](https://github.com/long-war-2/lwotc/files/3380703/save_relay.zip)
